### PR TITLE
feat: mapbox integration

### PR DESCRIPTION
**Pull request by edward-w-eth**
#2
2d map integration

### CODE REVIEW

File: chat/components/mapbox-map.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import { useState, useEffect, useRef } from 'react';
+import mapboxgl from 'mapbox-gl';
+
+export const Mapbox: React.FC = () => {
+  const mapContainer = useRef<HTMLDivElement>(null);
+  const [is3D, setIs3D] = useState(false);
+
+  useEffect(() => {
+    mapboxgl.accessToken = 
+      'pk.eyJ1IjoidG1hbG9uZXkiLCJhIjoiY2x3ZnF6MTR3MDBvczJrczdycHRldzAzMCJ9.o492x_ZQTRZiuqt8-r6YxA';
```
Comment (ngoiyaeric) [inline chat/components/mapbox-map.tsx:12]: For security purposes use environment variables to manage access tokens.

**Review by ngoiyaeric — CHANGES_REQUESTED** [review]
The logic looks ok, for security purposes abstract out the mapbox access token as an environment variable.

**Comment by edward-w-eth** [comment]
Thanks. EreQ
I will complete within 10 mins.
Warm regards
Edward Martin

**Comment by ngoiyaeric** [comment]
Make sure to test scrolling chats independent of mapbox static render.

**Comment by edward-w-eth** [comment]
Thanks, I will!
Access token issue fixed.

**Review by ngoiyaeric — APPROVED** [review]

**Comment by edward-w-eth** [comment]
Awesome. Thanks.

**Pull request merged**